### PR TITLE
Remove kirby metadata from protocol docs - Partially closes #37

### LINKED ISSUE #37: Remove kirby metadata from docs pages
A lot kirby CMS specific metadata in the documentation pages needs to be removed.

Except of the content section, everything should be removed.

- [x] Home
- [x] Protocol
- [x] Core
- [x] Elements
- [x] Commander
- [x] Explorer
- [x] Hub

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #37 for lisk protocol

### CODE REVIEW

**Review by janhack — DISMISSED** [review]

File: deprecated-transactions/deprecated-transactions.md [hunk 1]
```diff
@@ -152,21 +124,3 @@ The following is a representation of the resulting JSON object that will be broa
     ...
 }
 ```
-
-The final size of the transaction, with the signature, is 157 bytes. With a second signature it will be 221 bytes.
```
Comment (IkerAlus) [inline deprecated-transactions/deprecated-transactions.md]: I think this sentence should not be removed.

**Review by IkerAlus — COMMENTED** [review]
It is not a big issue since we do not use that file anymore but I think that sentence of "Deprecated Transactions" should not be deleted

File: deprecated-transactions/deprecated-transactions.md (hunk 1 above)
Comment (janhack) [reply deprecated-transactions/deprecated-transactions.md]: Agreed, I missed that one.

Comment (Tschakki) [reply deprecated-transactions/deprecated-transactions.md]: Good eye, sentence is restored now :)

**Review by IkerAlus — APPROVED** [review]

**Review by janhack — APPROVED** [review]

**Pull request merged**